src/components/DatasetCreateView.jsx

```diff
@@ -113,73 +120,152 @@ export class DatasetCreateView extends React.PureComponent {
         </p>
         <Form model="dataset" onSubmit={data => this.handleSubmit(data)}>
           <FormRow>
-            <label htmlFor="dataset.name">Dataset name:</label>
             <Control.text
               model="dataset.name"
               id="dataset.name"
               required
               validators={{ name: this.validateName }}
+              component={(props) =>
+                <TextField
+                  {...props}
+                  label="Dataset Name"
+                  defaultValue={this.props.model}
+                  name={this.props.model}
+                  style={{width: '300px'}}
+                  variant="outlined"
+                />
+              }
             />
-            <Control.text
-              model="dataset.otherName"
-              id="dataset.otherName"
-              required
-              validators={{ name: this.validateName }}
-            />
-            <span> (this is the other name)</span>
-          </FormRow>
-
-          <FormRow>
-            <label htmlFor="dataset.description">Dataset description:</label>
-            <Control.textarea model="dataset.description" id="dataset.description" required />
           </FormRow>
 
           <FormRow>
-            <label htmlFor="dataset.readers">Access:</label>
-            <Control.text id="dataset.readers" model="dataset.readers" validators={{ isEmail }} />
+            <Control.textarea
+              model="dataset.description"
+              id="dataset.description"
+              required
+              component={(props) =>
+                <TextField
+                  {...props}
+                  defaultValue={this.props.model}
+                  style={{width: '800px'}} //fullWidth
```

Comment:
I'm concerned about hardcoding pixel values here. Would like to see if we can find an approach following the flex model (out of the scope of this PR).